index.bs

```diff
@@ -2405,6 +2554,17 @@ To <dfn noexport>parse source-registration JSON</dfn> given a [=byte sequence=]
 1. Let |epsilon| be the user agent's [=max settable event-level epsilon=].
 1. Set |epsilon| to |value|["<code>[=source-registration JSON key/event_level_epsilon=]</code>"] if it [=map/exists=]:
 1. If |epsilon| is not a double, is less than 0, or is greater than the user agent's [=max settable event-level epsilon=], return null.
+1. Let |aggregatableDebugBudget| be 0.
+1. Let |aggregateDebugReportingConfig| be a new [=aggregate debug reporting config=].
+1. If |value|["<code>[=source-registration JSON key/aggregate_debug_reporting=]</code>"] [=map/exists=]:
+    1. Let |aggregateDebugReportingResult| be the result of running
+        [=parse source aggregate debug reporting config=] with
+        |value|["<code>[=source-registration JSON key/aggregate_debug_reporting=]</code>"],
+        |aggregatableDebugBudget|, and |aggregateDebugReportingConfig|.
+    1. Set |aggregatableDebugBudget| to |aggregateDebugReportingResult|[0].
+    1. Set |aggregateDebugReportingConfig| to |aggregateDebugReportingResult|[1].
+1. Let |aggregatableAttributionBudget| be [=allowed aggregatable budget per source=]
+    - |aggregatableDebugBudget|.
```

Comment:
```suggestion
1. Let |aggregatableAttributionBudget| be [=allowed aggregatable budget per source=] - |aggregatableDebugBudget|.
```

Otherwise it is rendered as a nested list item.